docs/jwst/tweakreg/README.rst

```diff
@@ -86,7 +86,8 @@ models to the custom catalog file name, the ``tweakreg_step`` also supports two
 other ways of supplying custom source catalogs to the step:
 
 1. Adding ``tweakreg_catalog`` attribute to the ``members`` of the input ASN
-   table - see `~jwst.datamodels.ModelContainer` for more details.
+   table - see `~jwst.datamodels.ModelLibrary` for more details.
+   # FIXME: does this still work as described?
```

Comment:
it can be removed, this does work and there is unit test coverage for it [here](https://github.com/spacetelescope/jwst/blob/582dbd953ac9dcba1b353dfcc26646913ccfeecb/jwst/tweakreg/tests/test_tweakreg.py#L261)